docs/development/build.md

```diff
@@ -14,24 +14,18 @@ After setting up Go, you should be able to `go get` cAdvisor as expected (we use
 $ go get -d github.com/google/cadvisor
```

Comment:
There's conflicting information above: "Ensure that your version of Go is at least 1.3." and "**Note**: cAdvisor requires Go 1.5 to build.". Could you fix it? Does the vendor directory work by default on go 1.5 or do you need to set a flag?